invenio_app_rdm/static_pages/static_pages.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
minor: in the context of the comments below, it might be better to name this `fixtures/pages.py` or similar.

Reply:
+1 to this!